ztag/schema.py

```diff
@@ -186,25 +201,53 @@ class CensysString(WhitespaceAnalyzedString):
         }),
         "subject_alt_name":alternate_name,
         "issuer_alt_name":alternate_name,
-        "crl_distribution_points":ListOf(URI()),
+        "crl_distribution_points":ListOf(URL()),
         "authority_key_id":HexString(),
         "subject_key_id":HexString(),
-        "extended_key_usage":ListOf(Signed32BitInteger()), # TODO: what sized integer should this be?
-        "certificate_policies":ListOf(CensysString()),
+        "extended_key_usage":SubRecord({
+            "value":ListOf(Signed32BitInteger()), # TODO: what sized integer should this be?
+            "server_auth":Boolean(doc="TLS WWW server authentication"),
+            "client_auth":Boolean(doc="TLS WWW client authentication"),
+            "code_signing":Boolean(doc="Signing of downloadable executable code"),
+            "email_protection":Boolean(doc="Email protection"),
+            "time_stamping":Boolean(doc="Binding the hash of an object to a time"),
+            "ocsp_signing":Boolean(doc="Signing OCSP responses")
```

Comment:
These tenses aren't all the same in the docstrings, but I'm bad at English and can't tell you what's correct. Probably something authoritative?